os/net/lwip/src/core/netif.c

```diff
@@ -503,10 +503,9 @@ void netif_set_default(struct netif *netif)
  */
 void netif_set_up(struct netif *netif)
 {
-	//LWIP_DEBUGF(NETIF_DEBUG, ("Entry"));
+	LWIP_DEBUGF(NETIF_DEBUG, ("Entry"));
 	if (!(netif->flags & NETIF_FLAG_UP)) {
-		netif->flags |= NETIF_FLAG_UP;
-
+		netif->d_flags |= IFF_UP;
```

Comment:
why do you change this? I think it's not related to qemu

Reply:
@pillip8282 : addressed this comment